examples/servers/mysql-saved-items/service/service.go

```diff
@@ -36,7 +36,7 @@ func (s *SavedItemsService) Prefix() string {
 // This method helps satisfy the server.Service interface.
 func (s *SavedItemsService) Middleware(h http.Handler) http.Handler {
 	// wrap the response with our GZIP Middleware
-	return gziphandler.GzipHandler(h)
+	return context.ClearHandler(gziphandler.GzipHandler(h))
```

Comment:
Does this mean that every service now needs to call `ClearHandler()`  in `Middleware`? Can we push this up into gizmo cause I'm pretty sure most people will forget...